runtime/domain.c

```diff
@@ -632,8 +628,7 @@ static void domain_create(uintnat initial_minor_heap_wsize,
   }
 
   domain_state->id = d->id;
```

Comment:
Shouldn't that assignment also get moved down to remain close to the assignment of `unique_id`?

Reply:
Possibly, but I was uncertain whether any of the intervening function calls, into various modules, might use `domain_state->id` (for instance, in logging), whereas `unique_id` is not used much in the runtime and it was easy to determine that it was safe to move.

Reply:
Ok, let's keep `id` assigment location unchanged for now.